fast64_internal/sm64/sm64_level_writer.py

```diff
@@ -445,6 +445,29 @@ def replaceSegmentLoad(levelscript, segmentName, command, changedSegment):
     changedLoad[1][2] = segmentName + "SegmentRomEnd"
 
 
+def replaceScriptLoads(levelscript, obj):
+    newFuncs = []
+    for jumpLink in levelscript.levelFunctions:
+        target = jumpLink[1][0]  # format is [macro, list[args], commnent]
+        if "script_func_global_" not in target:
+            newFuncs.append(jumpLink)
+            continue
+        scriptNum = int(re.findall("\d+", target)[-1])
```

Comment:
```suggestion
        scriptNum = int(re.findall(r"\d+", target)[-1])
```
so as to not rely on `'\d'` -> `'\\d'`